app/views/posts/index.html.erb

```diff
@@ -1,22 +1,30 @@
-<h1>All Posts by Users</h1>
 <main>
-  
-  <div>
+  <section>
+    <div class="d-flex">
+      <div class="user-image wrap">
+        <%= image_tag @user.photo, class: 'list_image', alt: 'photo'%>
+      </div>
+      <div class="d-flex spc-btw m-100 f-end p-1 wrap">
+        <h2><%= @user.name %></h2>
+        <p class="m-50">Number of posts: <%= @user.posts_counter || 0 %></p>
+      </div>
+    </div>
+  </section>
+  <section>
```

Comment:
- [ ] Please use the `recent_comments` method on the `Post` model to render 5 most recent comments in the `app/views/posts/index.html.erb` page. You are currently displaying all the comments.

Reply:
done @codepantha Thank you!!